tools/plugins/com.liferay.ide.server.core/src/com/liferay/ide/server/core/portal/BundlePublishFullAdd.java

```diff
@@ -171,7 +171,7 @@ private String _getBundleUrl(File bundleFile, String bsn) throws MalformedURLExc
 		URL url = uri.toURL();
 
 		if (bundlePathLowerCase.endsWith(".war")) {
-			bundleUrl = "webbundle:" + url.toExternalForm() + "?Web-ContextPath=/" + bsn;
+			bundleUrl = "'webbundle:" + url.toExternalForm() + "?Web-ContextPath=/" + bsn + "'";
```

Comment:
Don't do it here.  Do it where ever we send dthe 'install'  or 'update' commands through Gogo.

Reply:
ok